modules/uhttp/http.go

```diff
@@ -106,8 +106,8 @@ func newModule(
 		Port:    defaultPort,
 		Timeout: defaultTimeout,
 	}
-	log := ulog.Logger(context.Background()).With(zap.String("module", host.Name()))
-	if err := host.Config().Scope("modules").Get(host.Name()).PopulateStruct(&cfg); err != nil {
+	log := ulog.Logger(context.Background()).With(zap.String("module", host.ModuleName()))
```

Comment:
Suggestion: since there is no context, you can use `zap.L().With(...` instead of `ulog.Logger(ctx)`